ios/Podfile

```diff
@@ -33,7 +33,6 @@ target 'JitsiMeetSDK' do
     :path => config[:reactNativePath],
     :hermes_enabled => true,
     :fabric_enabled => false,
-    :new_arch_enabled => false,
```

Comment:
Shall we ne explicit? We are explicit about hermes.

Reply:
We can, why not.